yarn.lock

```diff
@@ -11261,6 +11263,13 @@ storybook-chromatic@^2.2.2:
     tree-kill "^1.1.0"
     uuid "^3.3.2"
 
+storybook-react-router@^1.0.8:
```

Comment:
Mmm yeah - not sure how to handle this as it's a dependency to the application...

Reply:
I noticed in Franklin's `package.json` we have several storybook dependencies:
```
  "@storybook/addon-actions": "^5.3.6",
  "@storybook/addons": "^5.3.6",
  "@storybook/api": "^5.3.6",
  "@storybook/components": "^5.3.6",
  "storybook-react-router": "^1.0.8",
```
that aren't listed in `devDependencies` (`@storybook/addon-info`, `@storybook/addon-links`, `@storybook/react` are).

If we put everything in `devDependencies` do you think this would prevent storybook installed in uniprot?